moped-editor/src/views/projects/projectView/ProjectComponents/TheMap.js

```diff
@@ -201,15 +265,6 @@ export default function TheMap({
     setBounds(newBounds.flat());
   };
 
-  // This is a temporary to get data into the map sources
```

Comment:
This object made seeing what data was going where so bye 👋 These are split out and go directly into the components that were separated out from `ComponentsSourcesAndLayers`